app/Controllers/Http/EventsController.ts

```diff
@@ -19,27 +21,47 @@ export default class EventsController {
 
   // post one event
   public async store({ request, response }: HttpContextContract) {
-    const payload = await request.validate(CreateEventValidator)
     // TODO set is_public if user.role equal moderator or admin
     // TODO create_email must not be handed over, must be taken over via the auth
-    const event = await Event.create(payload)
-    await event.refresh()
+
+    const cover = request.file('cover')!
+    console.log(cover)
+    const event = new Event()
+
+    // const payload = await request.validate(CreateEventValidator)
+    // const event = await Event.create(payload)
+    if (cover) {
+      event.cover = Attachment.fromFile(cover)
+      await cover.move(Application.tmpPath('uploads'))
```

Comment:
cover.move funktioniert glaube nicht, so richtig erklären kann ich mir das aber nicht. tmpPath kommt dann `'/var/folders/6c/szt79xt57dqfc35m33g6p3380000gp/T/cldn6wa690001jmkf5i7l4qa0'`das nicht richtig ist. ich habe das in einer anderen func getestet und da kommt bei `Application.tmpPath('uploads')`der richtige path.
´Attachment.fromFile(cover)`macht aber schon alles, also das bild wird im tmpPath gespeichert.